addons/adminMenu/XEH_postInit.sqf

```diff
@@ -1,5 +1,25 @@
 #include "script_component.hpp"
 
+[QGVAR(resetGear), {
+    params ["_unit"];
+    diag_log text format ["[POTATO] Reseting Gear on %1 [%2]", (name _unit), _unit];
+    if ((isNull _unit) || {!alive _unit} || {!local _unit}) exitWith {};
```

Comment:
The way we're executing this event, none of the above _shouldn't_ happen. We should log an error message on the exit.